torch/_ops.py

```diff
@@ -25,6 +24,79 @@ def dl_open_guard():
     if _SET_GLOBAL_FLAGS:
         sys.setdlopenflags(old_flags)
 
+class OpOverload:
```

Comment:
This class is also the one that will expose the tags when they are added right?

Reply:
possibly